spec/acceptance/pe_status_check_spec.rb

```diff
@@ -184,6 +185,12 @@ class {'pe_status_check':
       #         result = run_shell('facter -p pe_status_check.S0019')
       #         expect(result.stdout).to match(%r{false})
       #       end
+      it 'if S0020 conditions for false are met' do
+        run_shell('puppet resource service pe-puppetserver  ensure=stopped')
```

Comment:
this is turning off and on  puppet server and not console services, so this test will not check if s00020 works